compiler/aarch64/codegen/TreeEvaluatorTable.hpp

```diff
@@ -0,0 +1,748 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+/*
+ * This table is #included in a static table.
+ * Only Function Pointers are allowed.
+ */
```

Comment:
Can you create a `TR::TreeEvaluator::unImpOpEvaluator` like the other platforms do that essentially asserts?  Each IL opcode in this table should then map to that `TR::TreeEvaluator::unImpOpEvaluator` until it is actually implemented.  It is far easier to determine what evaluators are complete and what aren't by looking at a table annotated that way.